langchain-interpreter/README.md

```
@@ -0,0 +1,95 @@
+  afm-langchain-interpreter afm /app/agent.afm.md
+```
+
+## Testing
+
+```bash
+uv run pytest
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)